packages/front-end/package.json

```diff
@@ -37,6 +37,7 @@
     "dirty-json": "^0.9.2",
     "js-yaml": "^4.1.0",
     "json-stringify-pretty-compact": "^3.0.0",
+    "json2csv": "^5.0.7",
```

Comment:
@jdorn Is this still ok here, or do I need to rip it out of the package.json entirely? I'm not super knowledgeable about the differences between npm and yarn.